app/controllers/crawlers_controller.rb

```diff
@@ -0,0 +1,74 @@
+class CrawlersController < ApplicationController
+  before_action :set_crawler, only: [:show, :edit, :update, :destroy]
+
+  # GET /crawlers
+  # GET /crawlers.json
+  def index
+    @crawlers = Crawler.all
+  end
+
+  # GET /crawlers/1
+  # GET /crawlers/1.json
+  def show
+  end
+
+  # GET /crawlers/new
+  def new
+    @crawler = Crawler.new
+  end
+
+  # GET /crawlers/1/edit
+  def edit
+  end
+
+  # POST /crawlers
+  # POST /crawlers.json
+  def create
+    @crawler = Crawler.new(crawler_params)
+
+    respond_to do |format|
+      if @crawler.save
+        format.html { redirect_to @crawler, notice: 'Crawler was successfully created.' }
+        format.json { render :show, status: :created, location: @crawler }
+      else
+        format.html { render :new }
+        format.json { render json: @crawler.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /crawlers/1
+  # PATCH/PUT /crawlers/1.json
+  def update
+    respond_to do |format|
+      if @crawler.update(crawler_params)
+        format.html { redirect_to @crawler, notice: 'Crawler was successfully updated.' }
+        format.json { render :show, status: :ok, location: @crawler }
+      else
+        format.html { render :edit }
+        format.json { render json: @crawler.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /crawlers/1
+  # DELETE /crawlers/1.json
+  def destroy
+    @crawler.destroy
+    respond_to do |format|
+      format.html { redirect_to crawlers_url, notice: 'Crawler was successfully destroyed.' }
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.